benchmarks/benchmark_serving.py

```diff
@@ -433,25 +421,81 @@ async def benchmark(
         "request_throughput": metrics.request_throughput,
         "input_throughput": metrics.input_throughput,
         "output_throughput": metrics.output_throughput,
-        "mean_ttft_ms": metrics.mean_ttft_ms,
-        "median_ttft_ms": metrics.median_ttft_ms,
-        "std_ttft_ms": metrics.std_ttft_ms,
-        "p99_ttft_ms": metrics.p99_ttft_ms,
```

Comment:
percentiles are added in if-blocks below when the corresponding metric is specified by, e.g., `--percentile-metrics=ttft,itl`.